maro/simulator/scenarios/vm_scheduling/topologies/azure.2019.original/config.yml

```diff
@@ -0,0 +1,25 @@
+# Initial buffer time budget.
```

Comment:
Since we don't hold the dataset, maintain this topology in your local and remove it from this PR?